### ISSUE TITLE
Backslashes

### ISSUE THREAD

**Issue by janezdu** [qema/nanosite#3]
This hasn't actually hurt me yet, BuT as a Windows user I get exposed to backslash shenanigans quite frequently so

``` html
<a href='http://zhuangjindu.github.io/pages\artwork.html'>Artwork</a>
```

**Comment by janezdu** [comment]
Seems like the browser will automatically correct this when it renders so this is no big deal.

**Comment by qema** [comment]
This was a good catch. Fixed in 0.1.6.
